code/game/objects/structures/girders.dm

```diff
@@ -165,7 +165,7 @@
 					qdel(src)
 				return
 
-		if(S.sheettype != "runed")
+		if(S.sheettype != "runed" && S.sheettype != "bronze")
```

Comment:
Could you explain why this needs to exist?

Reply:
Because otherwise you can use bronze on a girder to make a bronze wall instead of the special snowflake bronze girders. The issue with building walls on special snowflake girders is that if you build it on a normal girder and then remove the wall, the wall is now the snowflake girder. I didnt bother changing the behavior much aside from fixing this issue in the same way as runed metal because runed metal has the same method of preventing snowflaking girders, so no consistency issue.

Reply:
Could this check be moved to the sheet somehow? Like a has_unique_girder bitflag or something? Duplicate checks like this can lead to sheettypes piling up, and I'd much prefer this as something more generic

Reply:
Set it back to draft while  the change request is done.